entry/src/main/java/me/dozen/sample/Test.java

```diff
@@ -0,0 +1,58 @@
+package me.dozen.sample;
+
+import ohos.app.Context;
+import ohos.hiviewdfx.HiLog;
+import ohos.hiviewdfx.HiLogLabel;
+import me.dozen.dpreference.DPreference;
+
+/**
+ * Created by wangyida on 15/12/18.
+ */
```

Comment:
This java doc can be removed as the file is developed from scratch

Reply:
Done